packages/insomnia-importers/src/importers/openapi-3.ts

```diff
@@ -110,6 +110,22 @@ const parseDocument = (rawData: string): OpenAPIV3.Document | null => {
   }
 };
 
+/**
+ * Checks if the given property name is an open-api extension
+ * @param property The property name
+ */
+const isSpecExtension = (property: string): boolean => {
```

Comment:
if you wanted to, we could make this a type guard like this:

```suggestion
const isSpecExtension = (property: string): property is SpecExtension => {
```

We'd just need to add

```ts
type SpecExtension = `x-${string}`;
```
on the line above the tsdoc for this function.

Reply:
added in 960c47514d47c362d0606e78cca3b4b6bbf7325e just to get this PR closer to merging